app/views/govuk_component/docs/metadata.yml

```diff
@@ -17,6 +17,15 @@ fixtures:
     from:
```

Comment:
Oh I see. You mean the first example is redundant? I left it in so there was an example of both a non-truncated one and a truncated one. Happy to remove if it's not necessary.